.php_cs.dist

```diff
@@ -2,6 +2,7 @@
 
 use PhpCsFixer\Config;
 use PhpCsFixer\Finder;
+use Mollie\PhpCodingStandards\PhpCsFixer\Rules;
```

Comment:
M comes before P, and PHP-CS-Fixer would've fixed it for you 😜 

(Same for the snippet in the readme)